Hudl.Mjolnir/Command/Command.cs

```diff
@@ -181,12 +188,22 @@ internal Command(string group, string name, string breakerKey, string poolKey, T
 
             _log = LogManager.GetLogger("Hudl.Mjolnir.Command." + _name);
 
+            if (TimeoutsIgnored)
+            {
+                _log.Debug("Creating command with timeout disabled.");
+            }
             var timeout = GetTimeoutConfigurableValue(_name).Value;
             if (timeout <= 0)
             {
-                timeout = (long)defaultTimeout.TotalMilliseconds;
+                timeout = (long) defaultTimeout.TotalMilliseconds;
+            }
+            else
+            {
+                _log.DebugFormat("Timeout configuration override for this command of {0}",timeout);
+                //I think it is sensible to apply this timeout to the command if it has been explicitly set in configuration. 
+                //Therefore setting TimeoutsFullyDisabled to false regardless
+                TimeoutsIgnored = false;
```

Comment:
I guess ignoreTimeouts should be fully self explanatory. I was in two minds as to what I was going to do with these per command timeouts. My thought was that it may give users the ability to apply the blanket ignore first, then if they were interested in tweaking the timeout for a particular command they could just work with the per-command config. 
I guess since the per-command config overrides the default anyway you can still get the same use case. 

I'll change it.